### PR TITLE
Add def simplifier

### PR DESCRIPTION
**Pull request by bpcot23**
In this PR, I both implement a text simplification model and establish a virtual environment for our project.

The text simplification model is stored under model/text_simplifier.py. It can be run using the shell script ~./run_simplifier.sh located in the med_jargon_explain_inator directory. This model has a "model" parameter that should be specified on line 249, as well as various function parameters that can be changed to alter where the sentences and evaluations are output (write new files to the directory, or just return printed dictionaries). There are three model options: BERT, Pegasus, and T5, with T5 currently getting the best evaluation results on our three evaluation metrics (METEOR, Rouge1, Rouge2).

Additionally, I implemented a setup folder which includes a requirements.yml folder, and two shell scripts used to create a virtual environment and to update the virtual environment. These play a key role in running the entire system, and text_simplifier.py needs to be run on the virtual environment to ensure the user has all the correct packages and versions installed.

### CODE REVIEW

File: BART_evaluations.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ROUGE1 score: AggregateScore(low=Score(precision=0.28437878595922683, recall=0.7859388888279921, fmeasure=0.40167933124097793), mid=Score(precision=0.3302329966344789, recall=0.839176344461914, fmeasure=0.4455972190531664), high=Score(precision=0.38074499496583225, recall=0.8866876333778365, fmeasure=0.4922773836004021))
```
Comment (jonjiang1) [inline BART_evaluations.txt]: Could we throw the evaluation and sentence text files into a subfolder in data? Let's try and keep the top level of the repo clean.

File: model/old_architecture.py [hunk 2]
```diff
@@ -0,0 +1,199 @@
+import torch
+from transformers import BertTokenizer, BertForSequenceClassification
+from transformers import BartForConditionalGeneration, BartTokenizer
+from transformers import T5Tokenizer, T5ForConditionalGeneration
+from torch.utils.data import Dataset, DataLoader
+from transformers import AdamW
+from tqdm import tqdm
+import pandas as pd
+
+class MedicalDataset(Dataset):
```
Comment (jonjiang1) [inline model/old_architecture.py:10]: Class comments please! A quick couple lines for what this class is used for is enough.

Comment (bpcot23) [reply model/old_architecture.py:10]: Done! Though I added it on the text_simplifier.py script instead of here

File: BART_evaluations.txt (hunk 1 above)
Comment (bpcot23) [reply BART_evaluations.txt]: Good idea, done

**Review by jonjiang1 — APPROVED** [review]

**Comment by jonjiang1** [comment]
@bpcot23 approved, LGTM!

**Pull request merged**